### PR TITLE
[Localization] - Missing translations to the Filter bar in some languages

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes?
Translations of elements of the filter bar of the starter selection screen.
(+ Pokerus truned into Pokérus with an accent in all languages, as it should be)

## Why am I doing these changes?
To have them translated

## What did change?
Entires translations in the filter bar

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/es/filter-bar.ts [hunk 1]
```diff
@@ -22,9 +22,9 @@ export const filterBar: SimpleTranslationEntries = {
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "pokerus": "Pokérus",
+  "hasPokerus": "Pokérus - Yes",
+  "noPokerus": "Pokérus - No",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:27]:
```suggestion
  "pokerus": "Pokérus",
  "hasPokerus": "Pokérus - Sí",
  "noPokerus": "Pokérus - No",
```

File: src/locales/es/filter-bar.ts [hunk 2]
```diff
@@ -22,9 +22,9 @@ export const filterBar: SimpleTranslationEntries = {
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:24]:
```suggestion
  "hiddenAbility": "Habilidad oculta",
  "hasHiddenAbility": "Habilidad oculta - Sí",
  "noHiddenAbility": "Habilidad oculta - No",
```

File: src/locales/es/filter-bar.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "Passive",
   "passiveUnlocked": "Pasiva Desbloq.",
   "passiveLocked": "Pasiva Bloq.",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
   "costReductionUnlocked": "Cost Reduction Unlocked",
   "costReductionLocked": "Cost Reduction Locked",
   "ribbon": "Ribbon",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:19]:
```suggestion
  "costReduction": "Reducción de coste",
  "costReductionUnlocked": "Reducción de coste desbloq.",
  "costReductionLocked": "Reducción de coste bloq.",
  "ribbon": "Lazo",
```

File: src/locales/es/filter-bar.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "Passive",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:13]:
```suggestion
  "passive": "Pasiva",
```

File: src/locales/es/filter-bar.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ export const filterBar: SimpleTranslationEntries = {
...
   "passiveUnlocked": "Pasiva Desbloq.",
   "passiveLocked": "Pasiva Bloq.",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:15]:
```suggestion
  "passiveUnlocked": "Pasiva desbloq.",
  "passiveLocked": "Pasiva bloq.",
```

**Review by AsdarDevelops — COMMENTED** [review]
Spanish done and fixed some other things in the file:

"costReductionUnlocked" might be too long so let me know if it needs shortening

File: src/locales/de/filter-bar.ts [hunk 6]
```diff
@@ -13,7 +13,7 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "Passive",
   "passiveUnlocked": "Passive freigeschaltet",
   "passiveLocked": "Passive gesperrt",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
   "costReductionUnlocked": "Cost Reduction Unlocked",
   "costReductionLocked": "Cost Reduction Locked",
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:18]:
```suggestion
  "costReduction": "Kostenreduzierung",
  "costReductionUnlocked": "Kostenreduzierung möglich",
  "costReductionLocked": "Kostenreduzierung nicht möglich",
```

File: src/locales/de/filter-bar.ts [hunk 7]
```diff
@@ -22,9 +22,9 @@ export const filterBar: SimpleTranslationEntries = {
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "pokerus": "Pokérus",
+  "hasPokerus": "Pokérus - Yes",
+  "noPokerus": "Pokérus - No",
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:27]:
```suggestion
  "pokerus": "Pokérus",
  "hasPokerus": "Pokérus - Ja",
  "noPokerus": "Pokérus - Nein",
```

**Comment by Adri1** [comment]
@CodeTappert You forgot the Hidden Abilities entries

File: src/locales/de/filter-bar.ts [hunk 8]
```diff
@@ -13,18 +13,18 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "costReductionLocked": "Cost Reduction Locked",
+  "costReduction": "Kostenreduzierung",
+  "costReductionUnlocked": "Kostenreduzierung möglich",
+  "costReductionLocked": "Kostenreduzierung nicht möglich",
   "ribbon": "Band",
   "hasWon": "Hat Klassik-Modus gewonnen",
   "hasNotWon": "Hat Klassik-Modus nicht gewonnen",
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:24]:
```suggestion
  "hiddenAbility": "Versteckte Fähigkeit",
  "hasHiddenAbility": "Versteckte Fähigkeit - Ja",
  "noHiddenAbility": "Versteckte Fähigkeit - Nein",
```

File: src/locales/de/filter-bar.ts [hunk 9]
```diff
@@ -13,21 +13,21 @@ export const filterBar: SimpleTranslationEntries = {
...
   "passiveLocked": "Passive gesperrt",
-  "costReduction": "Cost Reduction",
-  "costReductionUnlocked": "Cost Reduction Unlocked",
-  "costReductionLocked": "Cost Reduction Locked",
+  "costReduction": "Kostenreduzierung",
+  "costReductionUnlocked": "Kostenreduzierung möglich",
+  "costReductionLocked": "Kostenreduzierung nicht möglich",
   "favorite": "Favorite",
   "isFavorite": "Favorite - Yes",
   "notFavorite": "Favorite - No",
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:21]:
```suggestion
  "favorite": "Favorit",
  "isFavorite": "Favorit - Ja",
  "notFavorite": "Favorit - Nein",
```

File: src/locales/pt_BR/filter-bar.ts [hunk 10]
```diff
@@ -16,7 +16,7 @@ export const filterBar: SimpleTranslationEntries = {
   "costReduction": "Redução de Custo",
   "costReductionUnlocked": "Redução de Custo Desbloq.",
   "costReductionLocked": "Redução de Custo Bloq.",
-  "favorite": "Favorite",
+  "favorite": "Favorite",//To be translated
   "isFavorite": "Favorite - Yes",
   "notFavorite": "Favorite - No",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/filter-bar.ts:21]:
```suggestion
  "favorite": "Favoritos",
  "isFavorite": "Favoritos - Sim",
  "notFavorite": "Favoritos - Não",
```

File: src/locales/zh_CN/filter-bar.ts [hunk 11]
```diff
@@ -16,7 +16,7 @@ export const filterBar: SimpleTranslationEntries = {
   "costReduction": "费用降低",
   "costReductionUnlocked": "已降费",
   "costReductionLocked": "未降费",
-  "favorite": "Favorite",
+  "favorite": "Favorite",//To be translated
```
Comment (40chyan) [inline src/locales/zh_CN/filter-bar.ts:19]:
```suggestion
  "favorite": "最爱",//To be translated
```

File: src/locales/zh_CN/filter-bar.ts [hunk 12]
```diff
@@ -16,7 +16,7 @@ export const filterBar: SimpleTranslationEntries = {
   "costReduction": "费用降低",
   "costReductionUnlocked": "已降费",
   "costReductionLocked": "未降费",
-  "favorite": "Favorite",
+  "favorite": "最爱",
   "isFavorite": "Favorite - Yes",
   "notFavorite": "Favorite - No",
```
Comment (40chyan) [inline src/locales/zh_CN/filter-bar.ts:21]:
```suggestion
  "isFavorite": "包含最爱",
  "notFavorite": "不包含最爱",
```

**Review by EnochG1 — APPROVED** [review]

File: src/locales/zh_TW/filter-bar.ts [hunk 13]
```diff
@@ -25,9 +25,9 @@ export const filterBar: SimpleTranslationEntries = {
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "pokerus": "Pokérus",
+  "hasPokerus": "Pokérus - Yes",
+  "noPokerus": "Pokérus - No",
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:30]:
```suggestion
  "pokerus": "病毒",
  "hasPokerus": "有病毒",
  "noPokerus": "無病毒",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 14]
```diff
@@ -13,10 +13,10 @@ export const filterBar: SimpleTranslationEntries = {
...
   "passiveUnlocked": "被動解鎖",
   "passiveLocked": "被動未解鎖",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
   "costReductionUnlocked": "Cost Reduction Unlocked",
   "costReductionLocked": "Cost Reduction Locked",
-  "favorite": "Favorite",
+  "favorite": "Favorite",//Also here
   "isFavorite": "Favorite - Yes",
   "notFavorite": "Favorite - No",
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:21]:
```suggestion
  "isFavorite": "包含最愛",
  "notFavorite": "不包含最愛",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 15]
```diff
@@ -13,10 +13,10 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "被動",
   "passiveUnlocked": "被動解鎖",
   "passiveLocked": "被動未解鎖",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
   "costReductionUnlocked": "Cost Reduction Unlocked",
   "costReductionLocked": "Cost Reduction Locked",
-  "favorite": "Favorite",
+  "favorite": "Favorite",//Also here
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:19]:
```suggestion
  "favorite": "最愛",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 16]
```diff
@@ -13,10 +13,10 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "被動",
   "passiveUnlocked": "被動解鎖",
   "passiveLocked": "被動未解鎖",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
   "costReductionUnlocked": "Cost Reduction Unlocked",
   "costReductionLocked": "Cost Reduction Locked",
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:18]:
```suggestion
  "costReductionUnlocked": "已降費",
  "costReductionLocked": "未降費",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 17]
```diff
@@ -13,10 +13,10 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "被動",
   "passiveUnlocked": "被動解鎖",
   "passiveLocked": "被動未解鎖",
-  "costReduction": "Cost Reduction",
+  "costReduction": "Cost Reduction",//Also here
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:16]:
```suggestion
  "costReduction": "費用降低",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 18]
```diff
@@ -25,9 +25,9 @@ export const filterBar: SimpleTranslationEntries = {
   "hiddenAbility": "Hidden Ability",
   "hasHiddenAbility": "Hidden Ability - Yes",
   "noHiddenAbility": "Hidden Ability - No",
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:27]:
```suggestion
  "hiddenAbility": "隱藏特性",
  "hasHiddenAbility": "有隱藏特性",
  "noHiddenAbility": "無隱藏特性",
```

**Review by mercurius-00 — COMMENTED** [review]
zh-tw

File: src/locales/pt_BR/filter-bar.ts [hunk 19]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "Passiva",
   "passiveUnlocked": "Passiva Desbloqueada",
   "passiveLocked": "Passiva Bloqueada",
-  "passiveUnlockable": "Passive - Can Unlock",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
```
Comment (josericardo-fo) [inline src/locales/pt_BR/filter-bar.ts:16]:
```suggestion
  "passiveUnlockable": "Passiva - Pode Desbloquear",
```

File: src/locales/pt_BR/filter-bar.ts [hunk 20]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
   "costReduction": "Redução de Custo",
   "costReductionUnlocked": "Redução de Custo Desbloq.",
   "costReductionLocked": "Redução de Custo Bloq.",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "costReductionUnlockable": "Cost Reduction - Can Unlock",//Add translation here
```
Comment (josericardo-fo) [inline src/locales/pt_BR/filter-bar.ts:20]:
```suggestion
  "costReductionUnlockable": "Redução de Custo - Pode Desbloquear",
```

File: src/locales/pt_BR/filter-bar.ts [hunk 21]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
   "ribbon": "Fita",
   "hasWon": "Fita - Sim",
   "hasNotWon": "Fita - Não",
   "hiddenAbility": "Habilidade Oculta",
   "hasHiddenAbility": "Habilidade Oculta - Sim",
   "noHiddenAbility": "Habilidade Oculta - Não",
-  "egg": "Egg",
-  "eggPurchasable": "Purchasable Egg",
+  "egg": "Egg",//Add translation here
+  "eggPurchasable": "Purchasable Egg",//Add translation here
```
Comment (josericardo-fo) [inline src/locales/pt_BR/filter-bar.ts:31]:
```suggestion
  "egg": "Ovo",
  "eggPurchasable": "Ovo Comprável",
```

File: src/locales/es/filter-bar.ts [hunk 22]
```diff
@@ -3,35 +3,35 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
...
+  "isFavorite": "Favoritos - Sí",
+  "notFavorite": "Favoritos - No",
+  "ribbon": "Cinta",
+  "hasWon": "Cinta - Sí",
+  "hasNotWon": "Cinta - No",
+  "hiddenAbility": "Habilidad oculta",
+  "hasHiddenAbility": "Habilidad oculta - Sí",
+  "noHiddenAbility": "Habilidad oculta - No",
+  "egg": "Egg",//Add translation here
+  "eggPurchasable": "Purchasable Egg",//Add translation here
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:31]:
```suggestion
  "egg": "Huevo",
  "eggPurchasable": "Huevo obtenible",
```

File: src/locales/es/filter-bar.ts [hunk 23]
```diff
@@ -3,35 +3,35 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
...
-  "noHiddenAbility": "Hidden Ability - No",
-  "egg": "Egg",
-  "eggPurchasable": "Purchasable Egg",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "passive": "Pasiva",
+  "passiveUnlocked": "Pasiva desbloq.",
+  "passiveLocked": "Pasiva bloq.",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:16]:
```suggestion
  "passiveUnlockable": "Pasiva - puede desbloq.",
```

**Review by AsdarDevelops — COMMENTED** [review]
I that's it. Let me know if I missed anything (Spanish)

File: src/locales/es/filter-bar.ts [hunk 24]
```diff
@@ -3,35 +3,35 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
...
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "passive": "Pasiva",
+  "passiveUnlocked": "Pasiva desbloq.",
+  "passiveLocked": "Pasiva bloq.",
+  "passiveUnlockable": "Pasiva - puede desbloq.",
+  "costReduction": "Reducción de coste",
+  "costReductionUnlocked": "Reducción de coste desbloq.",
+  "costReductionLocked": "Reducción de coste bloq.",
+  "costReductionUnlockable": "Cost Reduction - Can Unlock",//Add translation here
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:20]:
```suggestion
  "costReductionUnlockable": "Red. de coste - puede desbloq.",
```

File: src/locales/es/filter-bar.ts [hunk 25]
```diff
@@ -3,35 +3,35 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
...
+  "isFavorite": "Favoritos - Sí",
+  "notFavorite": "Favoritos - No",
+  "ribbon": "Cinta",
+  "hasWon": "Cinta - Sí",
+  "hasNotWon": "Cinta - No",
+  "hiddenAbility": "Habilidad oculta",
+  "hasHiddenAbility": "Habilidad oculta - Sí",
+  "noHiddenAbility": "Habilidad oculta - No",
+  "egg": "Huevo",
+  "eggPurchasable": "Huevo obtenible",
```
Comment (AsdarDevelops) [inline src/locales/es/filter-bar.ts:31]:
```suggestion
  "eggPurchasable": "Huevo - puede comprar",
```

**Review by AsdarDevelops — COMMENTED** [review]
Was missing one. Changed another line to be more fitting with the other two. Personally, I recommend doing the same thing on the other locales, imo it fits better.

File: src/locales/de/filter-bar.ts [hunk 26]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
   "passiveLocked": "Passive gesperrt",
-  "passiveUnlockable": "Passive - Can Unlock",
-  "costReduction": "Cost Reduction",
-  "costReductionUnlocked": "Cost Reduction Unlocked",
-  "costReductionLocked": "Cost Reduction Locked",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:16]:
```suggestion
  "passiveUnlockable": "Passive - Freischalten möglich",
```

File: src/locales/de/filter-bar.ts [hunk 27]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "egg": "Egg",
-  "eggPurchasable": "Purchasable Egg",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "hiddenAbility": "Versteckte Fähigkeit",
+  "hasHiddenAbility": "Versteckte Fähigkeit - Ja",
+  "noHiddenAbility": "Versteckte Fähigkeit - Nein",
+  "egg": "Egg",//Add translation here
+  "eggPurchasable": "Purchasable Egg",//Add translation here
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:31]:
```suggestion
  "egg": "Ei",
  "eggPurchasable": "Kauf möglich",

File: src/locales/de/filter-bar.ts [hunk 28]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "costReductionLocked": "Cost Reduction Locked",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
+  "costReduction": "Kostenreduzierung",
+  "costReductionUnlocked": "Kostenreduzierung möglich",
+  "costReductionLocked": "Kostenreduzierung nicht möglich",
+  "costReductionUnlockable": "Cost Reduction - Can Unlock",//Add translation here
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:20]:
```suggestion
  "costReductionUnlockable": "Kostenreduzierung - Freischalten möglich",
```

File: src/locales/de/filter-bar.ts [hunk 29]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "costReductionUnlocked": "Cost Reduction Unlocked",
-  "costReductionLocked": "Cost Reduction Locked",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
+  "costReduction": "Kostenreduzierung",
+  "costReductionUnlocked": "Kostenreduzierung möglich",
+  "costReductionLocked": "Kostenreduzierung nicht möglich",
```
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts:19]:
```suggestion
  "costReductionUnlocked": "Kostenreduzierung noch möglich",
  "costReductionLocked": "Kostenreduzierung nicht mehr möglich",
```

File: src/locales/de/filter-bar.ts (hunk 28 above)
Comment (CodeTappert) [inline src/locales/de/filter-bar.ts]:
```suggestion
  "costReductionUnlocked": "Kosten bereits reduziert",
  "costReductionLocked": "Kosten noch nicht reduziert",
  "costReductionUnlockable": "Kosten können reduziert werden",
```

File: src/locales/pt_BR/filter-bar.ts [hunk 30]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "passiveUnlockable": "Passive - Can Unlock",
+  "passiveUnlockable": "Passiva - Pode Desbloquear",
   "costReduction": "Redução de Custo",
   "costReductionUnlocked": "Redução de Custo Desbloq.",
   "costReductionLocked": "Redução de Custo Bloq.",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "costReductionUnlockable": "Redução de Custo - Pode Desbloquear",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/filter-bar.ts:20]:
```suggestion
  "costReductionUnlockable": "Redução de Custo Disponível",
```

File: src/locales/zh_CN/filter-bar.ts [hunk 31]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "passiveUnlockable": "Passive - Can Unlock",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
   "costReduction": "费用降低",
   "costReductionUnlocked": "已降费",
   "costReductionLocked": "未降费",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "costReductionUnlockable": "Cost Reduction - Can Unlock",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_CN/filter-bar.ts:20]:
```suggestion
  "costReductionUnlockable": "可降费",
```

File: src/locales/zh_CN/filter-bar.ts [hunk 32]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "被动",
   "passiveUnlocked": "被动解锁",
   "passiveLocked": "被动未解锁",
-  "passiveUnlockable": "Passive - Can Unlock",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_CN/filter-bar.ts:16]:
```suggestion
  "passiveUnlockable": "被动可解锁",
```

File: src/locales/zh_CN/filter-bar.ts [hunk 33]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
+  "notFavorite": "不包含最爱",
   "ribbon": "缎带",
   "hasWon": "有缎带",
   "hasNotWon": "无缎带",
   "hiddenAbility": "梦特",
   "hasHiddenAbility": "有梦特",
   "noHiddenAbility": "无梦特",
-  "egg": "Egg",
-  "eggPurchasable": "Purchasable Egg",
+  "egg": "Egg",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_CN/filter-bar.ts:30]:
```suggestion
  "egg": "蛋",
```

File: src/locales/zh_CN/filter-bar.ts [hunk 34]
```diff
@@ -13,22 +13,22 @@ export const filterBar: SimpleTranslationEntries = {
...
+  "eggPurchasable": "Purchasable Egg",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_CN/filter-bar.ts:31]:
```suggestion
  "eggPurchasable": "可购买蛋",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 35]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "egg": "Egg",
-  "eggPurchasable": "Purchasable Egg",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "hiddenAbility": "隱藏特性",
+  "hasHiddenAbility": "有隱藏特性",
+  "noHiddenAbility": "無隱藏特性",
+  "egg": "Egg",//Add translation here
+  "eggPurchasable": "Purchasable Egg",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:31]:
```suggestion
  "egg": "蛋",
  "eggPurchasable": "可購買蛋",
```

File: src/locales/zh_TW/filter-bar.ts [hunk 36]
```diff
@@ -13,25 +13,25 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "costReductionLocked": "Cost Reduction Locked",
-  "costReductionUnlockable": "Cost Reduction - Can Unlock",
-  "favorite": "Favorite",
-  "isFavorite": "Favorite - Yes",
-  "notFavorite": "Favorite - No",
+  "passiveUnlockable": "Passive - Can Unlock",//Add translation here
+  "costReduction": "費用降低",
+  "costReductionUnlocked": "已降費",
+  "costReductionLocked": "未降費",
+  "costReductionUnlockable": "Cost Reduction - Can Unlock",//Add translation here
```
Comment (mercurius-00) [inline src/locales/zh_TW/filter-bar.ts:20]:
```suggestion
  "passiveUnlockable": "被動可解鎖",
  "costReduction": "費用降低",
  "costReductionUnlocked": "已降費",
  "costReductionLocked": "未降費",
  "costReductionUnlockable": "可降費",
```

**Review by mercurius-00 — COMMENTED** [review]
zh

**Review by josericardo-fo — APPROVED** [review]

**Pull request merged**